tools/pull-mdi-icons.js

```
@@ -51,32 +57,32 @@
+		log(`Requested version: ${underline(version)} ✔️`);
 
 		// Download CSS file (in dist/css/), woff2 webfont (in dist/fonts/)
 		const escapedVersion = version.replace(".", `\\.`);
 		const webfontZipPath = `${workspace}/${flavour}-webfont.zip`;
 		const webfontExtractedZipPath = `${workspace}/${flavour}-webfont`;
-		fs.mkdirSync(webfontExtractedZipPath);
+		mkdirSync(webfontExtractedZipPath);
 		try {
-			await upstream.downloadWebfontZip(webfontZipPath, flavour, version);
+			await downloadWebfontZip(webfontZipPath, flavour, version);
 		} catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)